BUILD.md

```diff
@@ -19,15 +19,15 @@
 
 Daffodil's build requirements include:
 
-* JDK 8 or higher
-* SBT 0.13.8 or higher
+* Java 8 or higher
+* sbt 0.13.8 or higher
```

Comment:
Do we really support sbt this old? I don't want to even bother checking it.

Reply:
I believe that version of sbt (or thereabouts) added the ability to use the version of sbt defined in project/build.properties. So as long as a user has that version or newer, it should work and download the correct version we actually build and test with. That said, 0.13 is really old, so if we want to bump it to something more modern that doesn't seem unreasonable.